test/smoke_test/smoke_test.py

```diff
@@ -9,4 +9,21 @@
 import torchaudio.sox_effects  # noqa: F401
 import torchaudio.transforms  # noqa: F401
 import torchaudio.utils  # noqa: F401
-from torchaudio.io import StreamReader  # noqa: F401
+
+
+def ffmpeg_test():
+    from torchaudio.io import StreamReader  # noqa: F401
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--no-ffmpeg", dest="ffmpeg", action="store_false")
+    parser.set_defaults(ffmpeg=True)
```

Comment:
I think `action="store_false"` will set the default value to `True`, so the line `parser.set_defaults(ffmpeg=True)` is redundant.

Reply:
Done